src/Navigation/styled.js

```diff
@@ -0,0 +1,80 @@
+import styled from "styled-components";
+import SearchIcon from "../Navigation/icons/search-icon.svg";
+
+export const Navigation = styled.header`
+  background-color: ${({ theme }) => theme.colors.headerBackground};
+  color: ${({ theme }) => theme.colors.white};
+  display: flex;
+  align-items: center;
+  justify-content: space-evenly;
+
+  @media (max-width: ${({ theme }) => theme.breakpoints.small}) {
+    flex-direction: column;
+    padding: 10px;
+  }
```

Comment:
This is unnecessary because you have the same styles in the media below and it covers screens up to medium breakpoint 😉

Reply:
Thanks, must be a left over from experimenting with styles!